functions/processQueueItem.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+
+const AWS = require("aws-sdk");
+const S3 = new AWS.S3({ apiVersion: "2006-03-01" });
+const SQS = new AWS.SQS({ apiVersion: "2012-11-05" });
+const request = require("request-promise-native");
+const { CONFIG_TABLE, QUEUE_NAME, CONTENT_BUCKET } = process.env;
+
+module.exports.handler = async function(
+  { MessageId, ReceiptHandle, Body },
+  context
+) {
+  const { requestId } = JSON.parse(Body);
+
+  console.log("MESSAGE BODY", requestId);
+
+  try {
+    const getResult = await S3.getObject({

```

Comment:
Here on the message processing side, the "image" content from S3 is fetched and deleted